cobra/main.go

```diff
@@ -13,8 +13,18 @@
 
 package main
 
-import "github.com/spf13/cobra/cobra/cmd"
+import (
+	"os"
+
+	"github.com/spf13/cobra/cobra/cmd"
+)
 
 func main() {
-	cmd.Execute()
+	if err := runMain(); err != nil {
+		os.Exit(1)
+	}
+}
+
+func runMain() error {
```

Comment:
Why did you add this function and do not use cmd.Execute directly?

Reply:
Corrected.